docs/docstring_template.md

```diff
@@ -0,0 +1,72 @@
+# Docstring Template
+--------------------
+
+Below is a docstring example. It contains a suggested docstring layout, example text and example MathJax.
+
+The layout below will render properly in Read the Docs. Please adjust the example text and MathJax as required.
+
+Note: backticks render differently in docstrings to elsewhere. Should you wish to include text in backticks, please use double backticks - e.g. ``scores.continuous.mean_error``. (If you only use one set of backticks, the text will instead be rendered as italics in Read the Docs). 
+
+```
+def function_name(*args, *, **kwargs) -> type hint:
+    '''
+    A short one or two line description of what the function does. 
+
+    Additional information (e.g. a paragraph or two) may be included here following the short description.
```

Comment:
Longer lines aren't wrapping on 228. I suggest limiting the line length so that it renders correctly